### ISSUE TITLE
Make table column widths more attractive

### ISSUE THREAD

**Issue by ptgott** [gravitational/docs#126]
Currently, table column widths in docs pages vary between docs sections and cut off long words. This is especially clear in the [metrics guide](https://github.com/gravitational/teleport/blame/2d66b8603304e115a2b0dee67a9f71bad346fccc/docs/pages/setup/reference/metrics.mdx#L70):

<img width="1207" alt="image" src="https://user-images.githubusercontent.com/25250668/184011028-5addc74e-7285-442f-bc94-eaa654a46649.png">

**Comment by alexfornuto** [comment]
To clarify what I think the issue (or at least part of it) is: in-line code blocks with a single word (no space) should not be broken up unless absolutely necessary. Currently the formatter breaks them very early in the line.

Possible solutions:

- Use a superset of Markdown (like [Markdown Extra](https://michelf.ca/projects/php-markdown/extra/#table) that supports additional formatting on tables.
- Allow the author to specify a style for the table, with pre-defined css styles to choose depending on which column we want to promote more width to.
- Allow the author to specify percentage of table for each column, and/or to disable mid-word breaking for that column. Not sure if/how that could be implemented.

**Comment by ninaTorgunakova** [comment]
I think it would be preferable to disable mid-word breaking for the code in the table columns. It will be working automatically and requires only small changes in CSS. Attached is a video showing this solution on pages with big tables and long code in the cells, including a metrics guide.

https://github.com/gravitational/docs/assets/78731462/271db444-7da1-440d-8d8d-20a4bfdc9602

But if someday it will be necessary to set some specific width for any column in a table, it can be done by editing the header in this way:
`| Command | Description |` -> `| <div style="width:150px">Command</div> | Description |`
It would work properly for the whole column.

**Comment by ptgott** [comment]
Thanks for taking a look at this!

> I think it would be preferable to disable mid-word breaking for the code in the table columns.

That sounds good to me and looks good so far in the video. I think it would be good to open a PR so we can inspect the preview site and see if there are any edge cases.

I'd prefer not to have to assign CSS styles directly within each page, since this makes the look and feel of the docs site harder to maintain or modify in the future.

**Comment by ninaTorgunakova** [comment]
@ptgott sure, I'll open the PR with this solution. No need to assign CSS styles within each page, changes will be only in the single component.

I checked this solution manually on most part of the pages with tables and also used regexp to find long strings of code inside the table cell. While checking, I found only one place where it looks not very nice:
![1](https://github.com/gravitational/docs/assets/78731462/81075bf0-bae3-49c1-95e6-ad01196b213b)
I noticed that in other places, such variables are written not in one block of code, but each in its own block of code, separated by commas. Looks like a typo here, so I separated these values in the content, and it becomes pretty:
![1-fix](https://github.com/gravitational/docs/assets/78731462/9af18ac3-0283-47a0-8c14-0636b2cff7c5)
Will make PRs with these fixes also.
